### DISCUSSION TITLE
Switch select items based on value of another select

### DISCUSSION THREAD

**Discussion by deanCompRSA** [vueform/vueform#370] [Questions]
Can I change the values of a select when another select value changes? 

Trying something along the lines of this but can't get it to work
https://stackblitz.com/edit/github-nzqvai-kzkwph?file=src%2FApp.vue

**Comment by adamberecz** [comment]
Yes, here are some examples:
- https://vueform.com/reference/select-element#string-endpoint
- https://vueform.com/recipes/dynamically-load-options-to-chained-select-boxes-from-endpoint
- https://vueform.com/recipes/dynamically-load-options-to-chained-select-boxes-with-async-functions
